src/cljs/org/broadinstitute/firecloud_ui/testrunner.cljs

```diff
@@ -0,0 +1,10 @@
+(ns org.broadinstitute.firecloud-ui.testrunner
+  (:require
+   cljs.test
+   org.broadinstitute.firecloud-ui.configtest
+   org.broadinstitute.firecloud-ui.main
+   org.broadinstitute.firecloud-ui.testutils
+   ))
+
+(defn run-all-tests []
+  (cljs.test/run-tests 'org.broadinstitute.firecloud-ui.configtest))
```

Comment:
It doesn't exist by default, though we could write it. I'd prefer to not do that for this initial version.